lldb/packages/Python/lldbsuite/test/tools/lldb-dap/dap_server.py

```diff
@@ -894,7 +894,8 @@ def request_launch(
         disableASLR=False,
         disableSTDIO=False,
         shellExpandArguments=False,
-        runInTerminal=False,
+        runInTerminal=False,  # deprecated
```

Comment:
Why not update/remove it if it's deprecated? We have control over the tests, we don't need any backward compatibility. If it's a matter of updating the existing tests in a separate PR then this is fine.